RELEASES.md

```diff
@@ -831,3 +831,16 @@
 11.8.1 2020-11-23
 -----------------
   - SMC-1482 - fix: redirected logs not in correct format
+
+11.9.0 2021-03-01
+-----------------
+ - SMC-1808: created initial test file
```

Comment:
we don't need the whole commit history here, just the summary of each ticket (feature added / bug fixed)

Reply:
- SMC-1808 - Add support for permissions on the happn user
- SMC-1897 - nedb data provider must sync db file after each persistent write